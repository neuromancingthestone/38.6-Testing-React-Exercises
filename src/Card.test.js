import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";


it("should render without crashing", () => {
  const photos = TEST_IMAGES;
  render(<Card 
    caption={photos.caption}
    src={photos.src}
    currNum={0}
    totalNum={3}
  />)
})

it("should match snapshot", () => {
  const photos = TEST_IMAGES;  
  const {asFragment} = render(<Card 
    caption={photos.caption}
    src={photos.src}
    currNum={0}
    totalNum={3}
  />)
  expect(asFragment()).toMatchSnapshot();
})