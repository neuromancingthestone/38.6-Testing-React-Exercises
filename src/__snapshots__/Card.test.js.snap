// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image 0 of 3.
    </small>
  </div>
</DocumentFragment>
`;
